### DISCUSSION TITLE
Displaying pdf within a dark mode scree

### DISCUSSION THREAD

**Discussion by pzorba75** [sumatrapdfreader/sumatrapdf#3929] [Documentation]
HI,
I'have been using sumatra-pdf for years and all pages where displayedon a white screen without any problem. And this is me préference.
Since a couple of hours, pdf pages are now displayed in darl mode. I'cannot go back to the previous mode.
I've reinstalled the latest version 3.5.2 64 b windows 10, in fact the same version as previously used.
The error continues.
Is there a workaround or a parameter to force sumatrapdf to display pdf pages in white mode?
Thanks by advance.
Pierre.

**Comment by GitHubRulesOK** [comment]
by default the inverse mode should not be there 
This should be default state of SumatraPDF 3.5.2 unless you use themes (second image) or press <kbd> I </kbd> (third image)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/36f1da9e-a67a-43db-abab-12e5d240437d)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7306deb0-ac59-4aeb-b312-b2b430358030)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/3b15e5fc-dba1-496a-b726-98899ba667d0)

**Reply by pzorba75** [reply]
Fine answer, pressing I will display what I'like. 
Have a good week-end.
Pierre
